distributed/shuffle/tests/test_shuffle.py

```diff
@@ -133,6 +133,7 @@ def test_raise_on_fuse_optimization():
             dd.shuffle.shuffle(df, "x", shuffle="p2p")
 
 
+@pytest.mark.xfail()
```

Comment:
This test doesn't make sense any more after these changes right? So xfail is wrong and it should be deleted?

Reply:
Yup, this is still work in progress; I've adjusted the test.